test/Scanner/MethodScannerTest.php

```diff
@@ -110,4 +111,27 @@ public function testMethodScannerWorksWithSingleAbstractFunction()
 
         self::assertTrue($method->isAbstract());
     }
+
+    /**
+     * @expectedException \Zend\Code\Exception\InvalidArgumentException
```

Comment:
Please use `$this->expectException()` instead, and right before the call that causes the exception to be thrown.